plugins/oauth/web_client/stylesheets/login.styl

```diff
@@ -29,15 +31,33 @@
     border-right 1px solid white
   .g-oauth-button-text
     display inline-block
-    padding 0 20px
+    text-align center
+    width 100px
```

Comment:
Actually just tested this out to make sure... it looks better as 109, so I'm going to change it to that. 

<img width="378" alt="screen shot 2015-11-12 at 11 24 44 am" src="https://cloud.githubusercontent.com/assets/555959/11123873/136c7006-8930-11e5-943e-aee687a73957.png">